subtrack.DAL/Entities/Subscription.cs

```diff
@@ -7,7 +7,7 @@ public class Subscription
     public int Id { get; set; }
 
     [Required]
-    [StringLength(12,ErrorMessage = "Service name should be less than 12 characters.")]
+    [StringLength(20,ErrorMessage = "Service name should be less than 20 characters.")]
```

Comment:
What was the max title length on sub details?

Reply:
Then we can set it to something like 50. You could probs just do the edit here on github directly, 50 here and 50 on the error message